.github/workflows/e2e-versions.yml

```
@@ -62,6 +62,32 @@ jobs:
         run: bash __tests__/verify-java.sh "${{ matrix.version }}" "${{ steps.setup-java.outputs.path }}"
         shell: bash
 
+  setup-java-alpine-linux:
+    name: ${{ matrix.distribution }} ${{ matrix.version }} (jdk-x64) - alpine-linux - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    container:
+      image: alpine:latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        distribution: ['temurin']
```

Hey, 
this is super cool. We have a similar logic for sapmachine, do you think it is feasible to add sapmachine to the distribution test array directly? 
```suggestion
        distribution: ['temurin','sapmachine']
```